### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

We removed the map on the main page to address boundary / border
concerns. In order to better maintain a appealing homepage, we then
adjusted the layout on the homepage to use multiple columns for the
countries and regions.

We also applied some minor adjustments to the layout throughout the
site for consistency and accessibility, adjusting text for clarity,
adding placeholder text to inputs, and moving the search inputs to be
inline with page text.

Connects #126 

### Demo

Screen Shot 2021-11-16 at 12 00 11 PM
Screen Shot 2021-11-16 at 12 00 23 PM
Screen Shot 2021-11-16 at 12 00 45 PM

## Testing Instructions

 * In the Netlify preview, view the countries on the homepage, and adjust the page width to confirm the columns respond reasonably at various screen sizes.  
 * Toggle to regions, and confirm that the search input disappears but the height of the region/country toggle remains the same. 
 * View the years page. The text should read 'survey year' instead of 'year(s)'. 
 * View the questions page. The search input should appear inline with the text at the top of the page. 
 * Confirm that the changes listed under ['How the survey contributes'](https://docs.google.com/document/d/178h9_bdBg2k5jiT1O8W50LAQJzMAvD1ydCv-z6nKBfw/edit#heading=h.cxvyw1914iu1) are present in the about modal. 
 * "Copyright © 2021 Facebook. All Rights Reserved." should be "Meta, Inc." instead of Facebook in the meta tags.
 * Confirm requested changes to the readme text:
     - "Facebook Gender Survey Dashboard" changed to "Gender Survey Dashboard from Meta"
     - "Interactive exploration of global results from the Facebook "Survey on Gender Equality At Home"" (changed Facebook to Meta) 
     - "The data used by this application is generated from a source set provided by Facebook's data team:" changed to "...provided by Data for Good at Meta."

### CODE REVIEW

File: src/app/src/components/FaqModal.jsx [hunk 1]
```diff
@@ -135,11 +135,7 @@ const FaqModal = ({ isOpen, onClose }) => {
                                 representative of the online population of the
                                 country/region, but not the total (online and
                                 offline) population. Please be mindful of this
-                                limitation when conducting your analysis. We
```
Comment (TaiWilkin) [inline src/app/src/components/FaqModal.jsx:138]: Samhir requested we remove this sentence.

**Comment by rajadain** [comment]
Taking a look now

File: src/app/src/components/AboutModal.jsx [hunk 2]
```diff
@@ -81,14 +81,14 @@ const AboutModal = ({ isOpen, onClose }) => {
                                     href='https://about.fb.com/news/2020/03/closing-the-gender-data-gap/'
                                     title='Project 17'
                                 />
-                                , two Facebook programs that use data to drive
+                                , two Meta programs that use data to drive
                                 progress on critical humanitarian issues. Data
                                 for Good provides tools built from
                                 privacy-protected data on Facebook’s platform,
```
Comment (rajadain) [inline src/app/src/components/AboutModal.jsx:87]: Should this stay "Facebook's platform" or be "Meta's platform"?

**Review by rajadain — APPROVED** [review]
+1 tested, this looks good! The search is nice and snappy too

File: src/app/src/components/AboutModal.jsx (hunk 2 above)
Comment (TaiWilkin) [reply src/app/src/components/AboutModal.jsx:87]: Meta's program but Facebook's platform - they collected the data on the Facebook application, but the program is run by Meta.

**Comment by TaiWilkin** [comment]
Thank you for reviewing!

**Pull request merged**